src/micromamba/install.cpp

```diff
@@ -373,15 +382,40 @@ parse_file_options()
 
             if (config["name"])
             {
-                env_name.add_rc_value(config["name"], file);
+                if (env_name.cli_configured())
+                {
+                    throw std::runtime_error(
```

Comment:
I think we need to allow that i think. can you check what conda does there?
IMO CLI should override yaml file here.

Reply:
This will be fixed by https://github.com/mamba-org/mamba/pull/816
Thanks for the review @wolfv